config/consensus.go

```diff
@@ -904,6 +908,9 @@ func initConsensusProtocols() {
 	vFuture := v27
 	vFuture.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
 
+	// Let the bytes value take more space. Key+Value is still limited to 128
+	vFuture.MaxAppBytesValueLen = 127
```

Comment:
This will likely become 128, once I have unit tests that show a blank key is a legal key.